descriptors/ModuleDescriptor-template.json

```diff
@@ -1013,7 +1013,7 @@
     },
     {
       "id": "feesfines",
-      "version": "15.0"
+      "version": "16.0"
```

Comment:
Do the changes associated with this version affect circulation's use of this API?

There are no code changes in mod-circulation, which suggests that they don't. If that is the case, please consider dual support for both `15.0` and `16.0`

Reply:
Done, thanks.